doc/SLSKPROTOCOL.md

```diff
@@ -1839,6 +1840,21 @@ The server returns a list of related search terms for a search query.
         1.  **string** <ins>term</ins>
         2.  **uint32** <ins>score</ins>
 
+## Server Code 160
+
+### ExcludedSearchPhrases
+
+The server sends a list of phrases not allowed on the search network. File paths containing such phrases must be excluded when responding to search requests.
```

Comment:
"`must be`" -> "`can then be`"

There is no technical reason as to why exclusion is mandatory.

Reply:
Changed the description.

As for your second suggestion, I don't feel comfortable deviating from SoulseekQt for now.